### ISSUE TITLE
Open source GetStatsByGroup

### ISSUE THREAD

**Issue by Sebasti1M** [Sebasti1M/BlazeServer#1]
I have to find it on the backup server. If someone _really_ wants it, just reply to this issue.

**Comment by MasterJochen** [comment]
Is it poible to modify the speed of gaining ranks in the current version? And if how can I do it. And do you have a official forum?

**Comment by MasterJochen** [comment]
And really ready great work man. This is just awesome working like a charm. You should also do a gta v online emu.

**Comment by SebastianMalek** [comment]
Thanks! No idea about stats, they seem to be very complex in the Blaze system.

**Comment by jeremychild** [comment]
Hello Sebastian I am refactoring your code to support BF4. Do you have Skype?

**Comment by SebastianMalek** [comment]
Hello! :)

My skype is maleksebastianm. I also use Discord:
https://discord.gg/0ns5t3QA7eJFC3qp

**Comment by SebastianMalek** [comment]
Oh and you'll have issues bypassing BF4/Warsaw DMR to hook gethostbyname()
and patching SSL verification. I never managed to complete it.

**Comment by squaredx** [comment]
Hey if you two want help, I got this code to work with BF4, and somewhat with Hardline. Im just having issues with the Component ID with HL.

**Comment by jeremychild** [comment]
Hi @squaredx and @SebastianMalek 

I have setup a private repo at gitlab.com

Please join it https://gitlab.com/cheesekun/blaze

**Comment by squaredx** [comment]
It's giving me a 404, I think you have to add me to the project. I have the same username there.

**Comment by jeremychild** [comment]
Add me on skype. My details should be on my github profile.

**Comment by SebastianMalek** [comment]
Yeah, same here. Logged in to GitLab with my GitHub account and I'm getting
a 404.

**Comment by jeremychild** [comment]
What is your username in gitlab?

**Comment by SebastianMalek** [comment]
SebastianMalek, same as here

**Comment by squaredx** [comment]
Could we possibly use discord, because I don't use Skype

**Comment by jeremychild** [comment]
Of course! Ill be on both for you :)

**Comment by squaredx** [comment]
Sweet do we already have a server, because that link Sebastian posted earlier is expired

**Comment by SebastianMalek** [comment]
Here's an invite code that won't expire: 0ns5t3QA7eMkd8pT
